### PR TITLE
Gwh apidocs 2045 euid js sdk cstg

### PR DESCRIPTION
**Pull request by genwhittTTD**
EUID JS SDK for CSTG.

### CODE REVIEW

File: api/v2/guides/publisher-client-side.md [hunk 1]
```diff
@@ -1,123 +1,315 @@
...
+If there was a problem generating the token, find the request in the **Network** tab. You can find the request by filtering for the string `client-generate`. Information about why the request failed should be available in the response.
+
+![Publisher Workflow](images/NetworkTraffic.png)
+
+
+## Example Code: Hashing and Base-64 Encoding
+
+The following code example demonstrates how to generate an email hash in JavaScript.
+
+(**GWH_AY for updated code example to exclude phone**)
```
[the remark this answers is not in the document]
Comment (lionell-pack-ttd) [reply api/v2/guides/publisher-client-side.md]: Yeah that code sample looks good.

File: api/v2/guides/publisher-client-side.md [hunk 2]
```diff
@@ -1,123 +1,311 @@
...
+## SDK for JavaScript Version
 
-For integration scenarios for publishers that do not use the [SDK for JavaScript](../sdks/client-side-identity.md), see [Publisher Integration Guide, Server-Only (Without SDK)](custom-publisher-integration.md). 
+Support for client-side token generation is available in version 3.2 and above of the SDK. 
 
-## Integration Steps 
+The URL for the SDK is:
 
-The following diagram outlines the steps required for a user to establish a EUID token with a publisher and how the EUID token integrates with the RTB bid stream.
+- [https://cdn.prod.uidapi.com/uid2-sdk-3.2.0.js](https://cdn.prod.uidapi.com/uid2-sdk-3.2.0.js)
```
Comment (lionell-pack-ttd) [inline api/v2/guides/publisher-client-side.md:28]: This URL should be: https://cdn.prod.euid.eu/euid-sdk-3.3.0.js

File: api/v2/guides/publisher-client-side.md [hunk 3]
```diff
@@ -1,123 +1,311 @@
...
 
-The following sections provide additional details for each step in the diagram:
- 
- 1. [Establish identity: user login](#establish-identity-user-login)
- 2. [Bid using EUID tokens](#bid-using-euid-tokens)
- 3. [Refresh tokens](#refresh-tokens)
- 4. [Clear Identity: user logout](#clear-identity-user-logout)
+If you want to use a debug build of the SDK, use the following URL instead:
+
+- [https://cdn.integ.uidapi.com/uid2-sdk-3.2.0.js](https://cdn.integ.uidapi.com/uid2-sdk-3.2.0.js)
```
Comment (lionell-pack-ttd) [inline api/v2/guides/publisher-client-side.md:34]: This URL should be: https://cdn.integ.euid.eu/euid-sdk-3.3.0.js

File: api/v2/guides/integration-javascript-server-side.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+| :--- | :--- | :--- |
+| 1-d | [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) | After the user authenticates and authorizes the creation of an EUID, use the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) endpoint to generate an EUID token using the email address provided by the user. Make sure it is normalized. |
+| 1-e | [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) | The endpoint returns an EUID token generated from the user's hashed or unhashed email address. |
+| 1-f | EUID SDK for JavaScript | The SDK sends the returned EUID token from step 1-e to the SDK in the `identity` property of its [init() function](../sdks/client-side-identity.md#initopts-object-void). |
+| 1-g | EUID SDK for JavaScript | Provide the SDK a callback function that will receive identity updates from the SDK and use them to initiate targeted advertising. |
+
+#### Example: JavaScript
+
+```js
+  window.__uid2 = window.__uid2 || {};
```
Comment (lionell-pack-ttd) [inline api/v2/guides/integration-javascript-server-side.md:67]: All instances of `__uid2` in this doc should instead be `__euid`. @genwhittTTD do you want one of the devs to do it, or are you comfortable with that?

Comment (lionell-pack-ttd) [reply api/v2/guides/integration-javascript-server-side.md:67]: The same thing applies to api/v2/guides/publisher-client-side.md.

File: api/v2/guides/publisher-client-side.md (hunk 2 above)
Comment (ashleysmithTTD) [reply api/v2/guides/publisher-client-side.md]: Done

File: api/v2/guides/publisher-client-side.md (hunk 3 above)
Comment (ashleysmithTTD) [reply api/v2/guides/publisher-client-side.md]: Done

**Review by ashleysmithTTD — APPROVED** [review]

**Pull request merged**